Makefile

```diff
@@ -3,20 +3,20 @@ test:
 testcov:
 	py.test --cov=ideascube/ --cov-report=term-missing
 develop:
-	pip install -r requirements-dev.txt
```

Comment:
I don't have a strong opinion about that, I did that because in my Dockerfile, I tried not to use a virtualenv. Feel free to revert this part if you think it's not a good idea.